packages/engine/Source/Core/Resource.js

```diff
@@ -765,7 +745,6 @@ Resource.prototype.clone = function (result) {
   result.proxy = this.proxy;
   result.retryCallback = this.retryCallback;
   result.retryAttempts = this.retryAttempts;
-  result._retryCount = 0;
```

Comment:
I think this is still needed, right?

Reply:
Yes, you're right. I was confusing it with the case with undefined `result`, where the `Resource` constructor already sets this value. But with a supplied `result`, we do need to reset it.